public/content/documentation/web/component/number-input.md

```diff
@@ -0,0 +1,219 @@
+</label>
+<input type="text"
+       id="pin"
+       aria-describedby="hint"
+       inputmode="numeric"
+       pattern="[0-9]*">
+<div id="hint" class="hint">
+  The pin number will expire after 1 hour
+</div>
+```
+<!-- TODO for all examples, this is allowing letters AND numbers despite pattern -->
+<example>
+   <label for="pin">
+   Pin number
+   </label>
+   <input type="text"
+         id="pin"
+         aria-describedby="hint"
+         inputmode="numeric"
+         pattern="[0-9]*">
+   <div id="hint" class="hint">
+   The pin number will expire after 1 hour
+   </div>
+</example>
+
+### Disabled and focusable number input (preferred)
+
+- Using the `aria-disabled` attribute will allow the input to be focusable and more discoverable
+
+```html
+<label for="security-id">
+  Security ID number
+</label>
+<input type="text"
+       id="security-id"
+       aria-describedby="security-id-hint"
+       inputmode="numeric"
+       pattern="[0-9]*"
+       aria-disabled="true">
+<div id="security-id-hint" class="hint">
+  The Security ID number will expire after 1 hour
+</div>
+```
+
+<example>
+   <label for="security-id">
+   Security ID number
+   </label>
+   <input type="text"
+         id="security-id"
+         aria-describedby="security-id-hint"
+         inputmode="numeric"
+         pattern="[0-9]*"
+         aria-disabled="true">
+   <div id="security-id-hint" class="hint">
+   The Security ID number will expire after 1 hour
+   </div>
+</example>
+
+### Fully disabled number input (avoid)
+
+- Fully `disabled` inputs are not focusable so may not be as discoverable in a form
+
+```html
+<label for="pin-disabled">
+  Pin number
+</label>
+<input type="text"
+       id="pin-disabled"
+       inputmode="numeric"
+       pattern="[0-9]*"
+       disabled>
+```
+
+<example>
+   <label for="pin-disabled">
+   Pin number
+   </label>
+   <input type="text"
+         id="pin-disabled"
+         inputmode="numeric"
+         pattern="[0-9]*"
+         disabled>
+</example>
+
+### Telephone number input
+
+- Setting type="tel" changes the keyboard for mobile app users
+
+```html
+<label for="phone">
+  Phone number
+</label>
+<input type="tel"
+       id="phone"
+       inputmode="numeric"
+       autocomplete="tel"
+       aria-describedby="hint-phone">
+<div class="hint" id="hint-phone">
+  We’ll never sell or share your information
+</div>
+```
+
+<example>
+   <label for="phone">
+   Phone number
+   </label>
+   <input type="tel"
+         id="phone"
+         inputmode="numeric"
+         autocomplete="tel"
+         aria-describedby="hint-phone">
+   <div class="hint" id="hint-phone">
+   We’ll never sell or share your information
+   </div>
+</example>
+
+## Developer notes
+
+- [Why the GOV.UK Design System team changed the input type for numbers](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/)
+- [Stepper/counter input example](/checklist-web/stepper-input/)
```

Comment:
does this URL need `/web-criteria/component/` in front of it?

Reply:
good catch!!